client/my-sites/plan-features/index.jsx

```diff
@@ -851,10 +872,21 @@ export default connect(
 									args.coupon = withDiscount;
 								}
 
+								if ( isUnlaunchedAndGoingAtomic ) {
+									if ( isInSignup ) {
```

Comment:
Just an observation: I don't think this `isInSignup` check is necessary because everywhere `isInSignup === true`, `onUpgradeClick` has also been provided, so it doesn't go down this branch of the ternary.

Reply:
Ya, this is probably unnecessary at the moment, but in case it changes it seems worth leaving the conditional to me as it communicates the intent at the cost of a boolean check.